### ISSUE TITLE
bug: redpanda helm release in false state

### ISSUE THREAD

**Issue by metacoma** [mindwm/mindwm-gitops#118]
`$ kubectl get redpandas.cluster.redpanda.com --all-namespaces`
```
NAMESPACE   NAME        READY   STATUS
redpanda    neo4j-cdc   False   HelmRelease 'redpanda/neo4j-cdc' is not ready
```

`kubectl -n flux-system logs deployment/helm-controller`
```json
{
  "namespace": "redpanda",
  "name": "neo4j-cdc",
  "reconcileID": "6a8407fe-e2d5-41f6-9dd0-4ddd39bfa460",
  "error": "template: redpanda/templates/console/configmap-and-deployment.yaml:67:4: executing \"redpanda/templates/console/configmap-and-deployment.yaml\" at <include \"_shims.render-manifest\" (list \"console.ConfigMap\" $wrappedSecretValues)>: error calling include: template: redpanda/templates/_shims.tpl:279:25: executing \"_shims.render-manifest\" at <include $tpl (dict \"a\" (list $dot))>: error calling include: template: redpanda/charts/console/templates/_configmap.go.tpl:13:80: executing \"console.ConfigMap\" at <tpl (toYaml $values.console.config) $dot>: error calling tpl: cannot retrieve Template.Basepath from values inside tpl function: kafka:\n  brokers:\n  - neo4j-cdc-0.neo4j-cdc.redpanda.svc.cluster.local.:9093\n  sasl:\n    enabled: false\n  schemaRegistry:\n    enabled: true\n    tls:\n      caFilepath: \"\"\n      certFilepath: \"\"\n      enabled: false\n      insecureSkipTlsVerify: false\n      keyFilepath: \"\"\n    urls:\n    - http://neo4j-cdc-0.neo4j-cdc.redpanda.svc.cluster.local.:8081\n  tls:\n    caFilepath: \"\"\n    certFilepath: \"\"\n    enabled: false\n    insecureSkipTlsVerify: false\n    keyFilepath: \"\"\nredpanda:\n  adminApi:\n    enabled: true\n    tls:\n      caFilepath: \"\"\n      certFilepath: \"\"\n      enabled: false\n      insecureSkipTlsVerify: false\n      keyFilepath: \"\"\n    urls:\n    - http://neo4j-cdc.redpanda.svc.cluster.local.:9644: \"BasePath\" is not a value"
}
```

**Comment by metacoma** [comment]
https://github.com/redpanda-data/redpanda-operator/commit/8b473ccf5ab3d5e9951669c2f12fe1fd96e59d43

**Comment by metacoma** [comment]
```
HelmChart 'redpanda/redpanda-neo4j-cdc' is not ready
reconcilation finished in 41.223744ms, next run in 1m0s
unable to update status after state update
unable to update status after reconciliation
Reconciler error
unable to update status after reconciliation
Reconciler error
reconcilation finished in 19.988305ms
reconcilation finished in 21.103859ms
reconcilation finished in 19.00325ms
unable to update status after reconciliation
Reconciler error
unable to update status after reconciliation
Reconciler error
reconcilation finished in 20.297694ms
```

**Comment by metacoma** [comment]
Related issue https://github.com/redpanda-data/redpanda/issues/12664

**Comment by metacoma** [comment]
Upstream issue: https://github.com/redpanda-data/redpanda-operator/issues/261

**Comment by metacoma** [comment]
resolved by #129 more details here https://github.com/redpanda-data/redpanda-operator/issues/261#issuecomment-2529723353
